packages/types/src/snap.ts

```diff
@@ -267,18 +264,59 @@ export type GetAddressResponseType<T> = T extends keyof GetAddressResponseTypeTa
   ? GetAddressResponseTypeTable[T]
   : never
 
-/**  TYPES USED WITH SIGNTRANSACTION() METHODS ** */
+/** TYPES USED WITH SENDTRANSACTION() METHODS */
+export type SendTransactionParams<T> = {
+  transaction: T
+  chainId?: string
+  origin?: string
+}
+
+export type AvalancheSendTransactionParams = SendTransactionParams<AvalancheSignTransaction>
+export type EthereumSendTransactionParams = SendTransactionParams<EthereumSignTransaction>
+
+type SendTransactionParamsTypeTable = {
+  [SupportedChainIds.AvalancheMainnet]: AvalancheSendTransactionParams
+  [SupportedChainIds.EthereumMainnet]: EthereumSendTransactionParams
+}
```

Comment:
Why only those two EVM chains, and similar to the comment above, are we ever in a flow where we send a transaction from the snap itself?

Reply:
`sendTransaction` is a combination sign/send as in the hdwallet-metamask adapter. Since these are the only chains that use the MetaMask Dapp API for signing instead of the hdwallet-native signer, we support `sendTransaction` methods for these instead of the combination of `signTransaction` and `broadcastTransaction`.

The snap has `broadcastTransaction` methods which make calls to the corresponding methods from Unchained, so transactions may be sent from the snap itself.